### ISSUE TITLE
Generate Exception: Access to the path '/su' is denied on Gentoo

### ISSUE THREAD

**Issue by svick** [dotnet/BenchmarkDotNet#882]
I am trying to run a trivial Benchmark on a Gentoo machine where I have limited access rights. When I do, I get the following output:

```
$ dotnet run -c Release
// Validating benchmarks:
// ***** BenchmarkRunner: Start   *****
// ***** Building 1 exe(s) in Parallel: Start   *****
// ***** Done, took 00:00:00 (0.13 sec)   *****
// Found benchmarks:
//   Program.M: DefaultJob

// Generate Exception: Access to the path '/su' is denied.

// ***** BenchmarkRunner: Finish  *****

// * Export *
  BenchmarkDotNet.Artifacts/results/Program-report.csv
  BenchmarkDotNet.Artifacts/results/Program-report-github.md
  BenchmarkDotNet.Artifacts/results/Program-report.html

// * Detailed results *
Program.M: DefaultJob
Runtime = ; GC =
There are not any results runs

// * Summary *

BenchmarkDotNet=v0.11.1, OS=gentoo
Intel Core i7-6700 CPU 3.40GHz (Skylake), 1 CPU, 8 logical and 4 physical cores
.NET Core SDK=2.1.401
  [Host] : .NET Core 2.1.3 (CoreCLR 4.6.26725.06, CoreFX 4.6.26725.05), 64bit RyuJIT


 Method | Mean | Error |
------- |-----:|------:|
      M |   NA |    NA |

Benchmarks with issues:
  Program.M: DefaultJob

// * Legends *
  Mean  : Arithmetic mean of all measurements
  Error : Half of 99.9% confidence interval
  1 ns  : 1 Nanosecond (0.000000001 sec)

// ***** BenchmarkRunner: End *****
Run time: 00:00:00 (0.13 sec), executed benchmarks: 0

// * Artifacts cleanup *
```

The directory `su` does exist in the filesystem root and I do not have the rights to access it:

```
$ ls /su
ls: cannot access '/su': Permission denied
```

I previously encountered an msbuild issue on the same machine (https://github.com/dotnet/cli/issues/7227), where msbuild was getting confused, because the existence of the directory `/m` made it assume command line argument with that value referred to that directory, instead of treating it as an option. Maybe the issue here is similar and could be also solved by using dashes instead of slashes for command line options?

For completeness, here is the benchmark and my environment info:

```
$ cat Program.cs
using System;
using BenchmarkDotNet.Attributes;
using BenchmarkDotNet.Running;

public class Program
{
    static void Main() => BenchmarkRunner.Run<Program>();

    [Benchmark]
    public void M() { }
}
$ cat bench.csproj
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp2.1</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="BenchmarkDotNet" Version="0.11.1" />
  </ItemGroup>

</Project>
$ dotnet --info
.NET Core SDK (reflecting any global.json):
 Version:   2.1.401
 Commit:    91b1c13032

Runtime Environment:
 OS Name:     gentoo
 OS Version:
 OS Platform: Linux
 RID:         gentoo-x64
 Base Path:   /afs/ms.mff.cuni.cz/u/o/ondep7am/diplomka/dotnet/sdk/2.1.401/

Host (useful for support):
  Version: 2.1.3
  Commit:  124038c13e

.NET Core SDKs installed:
  2.1.401 [/afs/ms.mff.cuni.cz/u/o/ondep7am/diplomka/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.3 [/afs/ms.mff.cuni.cz/u/o/ondep7am/diplomka/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.3 [/afs/ms.mff.cuni.cz/u/o/ondep7am/diplomka/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.3 [/afs/ms.mff.cuni.cz/u/o/ondep7am/diplomka/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```

**Comment by adamsitnik** [comment]
@svick thank you for a great bug report

This issue have been opened in 2018 and we still did not find the time to work on this...

I've recently performed a big study using BenchmarkDotNet and various Linux distros and hardware configurations and everything has worked fine (https://github.com/dotnet/runtime/issues/41871)

I think that I am just going to close the issue. If anyone can still reproduce it in 2020 with modern .NET Core and MSBuild and the latest BenchmarkDotNet version, please file a new issue.
